### ISSUE TITLE
Notmuch backend mailbox name case-sensitivity

### ISSUE THREAD

**Issue by jghauser** [soywod/himalaya#336]
When I do `himalaya list` in the terminal or `:Himalaya` in vim, I get the following error:

```
Error: cannot parse notmuch envelopes from query "all"

Caused by:
    0: cannot parse notmuch mail entry
    1: Invalid address found: must contain a '@' symbol
```
Is this because of a malformatted email? Or is it some problem with notmuch or the himalaya notmuch implementation? How could I troubleshoot it?

I'm on v0.5.8.

**Comment by soywod** [comment]
From the [wiki](https://github.com/soywod/himalaya/wiki/Tips:debug-and-logs), you can try to run `RUST_LOG=trace himalaya list` and see. This looks like an email address is not well formatted :disappointed:

**Comment by jghauser** [comment]
Thanks a lot! I've managed to find the offending email (I think!), but I don't see anything particularly strange with it. Can I contact you by email to send you the file, so you could have a look? I don't want to post it here as it probably contains some confidentialish information.

**Comment by soywod** [comment]
Sure, feel free to forward me the email at `clement.douin@posteo.net`.

**Comment by soywod** [comment]
The bug has been fixed in the [v0.6.0](https://github.com/soywod/himalaya/releases/tag/v0.6.0), it should work as expected. Feel free to reopen if you find it again!
